src/Service/ConfigReader.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the box project.
+ *
+ * (c) Anatoliy Melnikov <5785276@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Aeliot\PhpCsFixerBaseline\Service;
+
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+class ConfigReader
```

Comment:
1. Consider renaming it to `ConsoleOptionsReader` or something like this
2. Make class final